src/components/PrimaryNav.jsx

```diff
@@ -8,30 +8,52 @@ const PrimaryNav = () => {
   const { signIn, signOut, user } = useSupabaseAuth();
 
   return (
-    <nav className="flex bg-offWhite min-h-10 w-full font-roboto font-bold px-4 py-4 sm:py-2">
-      <div className="flex-1 flex text-2xl font-medium items-center">
+    <nav className="flex bg-offWhite min-h-10 w-full font-roboto font-bold px-2 sm:px-4 py-4 sm:py-2 items-center">
+      <div className="flex-1 flex items-center">
         <a href="https://opensauced.pizza">
           <img
-            className="h-7 mr-4"
+            className="h-7 mr-2 sm:mr-4 "
             alt="open
         sauced"
             src={logo}
           />
         </a>
       </div>
+      <div className="flex justify-center items-center space-x-2 sm:space-x-3">
+        <input
+          className=" ring-1 rounded-lg shadow-md w-[120px] h-full p-2 text-[9px] sm:text-xs sm:w-40 "
+          type="search"
+          placeholder="search open sauced 🔍"
+          id="repo-search"
+          name="repo-search"
+          aria-label="Search through repositories rendered out"
+        />
+        {!user && (
+          <div className="items-center">
+            <div
+              className="cursor-pointer"
+              onClick={async () => {
+                await signIn({ provider: 'github' });
+              }}
+            >
+              Login
+            </div>
+          </div>
+        )}
+      </div>
 
-      {!user && (
+      {/* {!user && (
```

Comment:
what is up with the comments here?

Reply:
for some weird reason, the logic appeared twice in my files. Maybe I will just remove it

Reply:
Yes, please. If it is needed we can add it back but currently it is dead code.

Reply:
I have also removed the code you highlighted here too